app/src/main/java/com/android/unio/ui/home/Home.kt

```diff
@@ -57,8 +56,7 @@ import kotlinx.coroutines.launch
 fun HomeScreen(
     navigationAction: NavigationAction,
     eventListViewModel: EventListViewModel = viewModel(factory = EventListViewModel.Factory),
-    onAddEvent: () -> Unit,
-    onEventClick: (Event) -> Unit
+    onAddEvent: () -> Unit
```

Comment:
Consider removing the `onAddEvent` parameter as it is not used, and will probably not be used in the future.